scipy/optimize/_numdiff.py

```diff
@@ -1,8 +1,11 @@
 """Routines for numerical differentiation."""
+
 from __future__ import division
 
 import numpy as np
 
+from ..sparse import issparse, csc_matrix, csr_matrix, lil_matrix, find
```

Comment:
No reason, I just thought it was more consistent to do relative imports in library code. Should I change it?